ambry-rest/src/main/java/com.github.ambry.rest/NettyMetrics.java

```diff
@@ -170,6 +171,8 @@ public NettyMetrics(MetricRegistry metricRegistry) {
     publicAccessLogRequestRate =
         metricRegistry.meter(MetricRegistry.name(PublicAccessLogHandler.class, "RequestArrivalRate"));
     healthCheckRequestRate = metricRegistry.meter(MetricRegistry.name(HealthCheckHandler.class, "RequestArrivalRate"));
+    channelStatusInconsistent =
```

Comment:
It seems like this may fit better as a counter, looking at other similar metrics (see line 107-113). However, they both should work about the same in practice, so if you prefer Meter keep it as is.